app/cdap/components/WranglerGrid/TransformationToolbar/iconStore.tsx

```diff
@@ -0,0 +1,225 @@
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+
+export const ColumnIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M12 3.5H19C19.5304 3.5 20.0391 3.71071 20.4142 4.08579C20.7893 4.46086 21 4.96957 21 5.5V19.5C21 20.0304 20.7893 20.5391 20.4142 20.9142C20.0391 21.2893 19.5304 21.5 19 21.5H12M12 3.5H5C4.46957 3.5 3.96086 3.71071 3.58579 4.08579C3.21071 4.46086 3 4.96957 3 5.5V19.5C3 20.0304 3.21071 20.5391 3.58579 20.9142C3.96086 21.2893 4.46957 21.5 5 21.5H12M12 3.5V21.5"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+export const StructureIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M3 4.5H21V7.79779"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M3 13H21"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M21 21.5004H3V17.9023"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+    <path
+      d="M12 4.5V21.2713"
+      stroke="#757575"
+      strokeWidth="2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+export const FragmentIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M14 11.5V10.5H13H4C3.44772 10.5 3 10.0523 3 9.5V4.5C3 3.94772 3.44772 3.5 4 3.5H20C20.5523 3.5 21 3.94772 21 4.5V20.5C21 21.0523 20.5523 21.5 20 21.5H15C14.4477 21.5 14 21.0523 14 20.5V11.5Z"
+      stroke="#757575"
+      strokeWidth="2"
+    />
+    <path
+      d="M3.01099 13.7969H10.6937V21.4796H4.01098C3.4587 21.4796 3.01099 21.0319 3.01099 20.4796V13.7969Z"
+      stroke="#757575"
+      strokeWidth="2"
+    />
+  </svg>
+);
+export const MathIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <rect x="3" y="2.5" width="18" height="20" rx="1" stroke="#757575" strokeWidth="2" />
+    <circle cx="8.51471" cy="13.8069" r="1.30694" fill="#757575" />
+    <circle cx="8.51471" cy="17.928" r="1.30694" fill="#757575" />
+    <rect x="7" y="5.68359" width="10.7704" height="3.11704" fill="#757575" />
+    <rect x="14.8462" y="12.5" width="2.07282" height="6.73595" rx="1.03641" fill="#757575" />
+  </svg>
+);
+export const SecurityIcon = (
+  <svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path
+      d="M12 22.5C12 22.5 20 18.5 20 12.5V5.5L12 2.5L4 5.5V12.5C4 18.5 12 22.5 12 22.5Z"
+      stroke="#757575"
+      strokeWidth="2.2"
+      strokeLinecap="round"
+      strokeLinejoin="round"
+    />
+  </svg>
+);
+export const OtherIcon = (
```

Comment:
Use Mui `MoreHoriz` icon?

Reply:
Yes, replaced it with import MoreHorizOutlinedIcon .